src/darwin/Framework/CHIPTests/MTRPairingTests.m

```diff
@@ -29,6 +29,7 @@
 // Fixture: chip-all-clusters-app --KVS "$(mktemp -t chip-test-kvs)" --interface-id -1 \
     --dac_provider credentials/development/commissioner_dut/struct_cd_origin_pid_vid_correct/test_case_vector.json \
     --product-id 32768 --discriminator 3839
+// CASE retry code paths can be tested using --faults chip_CASEServerBusy_f1 (or similar)
```

Comment:
But nothing i doing that right now, right?  This is for local, not CI testing?  Might be good to make this comment a bit less cryptic.

Reply:
Yes at the moment it's just a note for manual testing. Hopefully at some point we can integrate the fault injection cluster to include automated tests for this.